authors.js

```diff
@@ -41,7 +41,9 @@ export default [
   { username: 'iamstarkov',      start: '23 Feb 2015', first: '569802366297493505' },
   { username: 'filipovskii',     start: '09 Feb 2015', first: '564697062378594304' },
   { username: 'shuvalov_anton',  start: '03 Feb 2015', first: '562516792753811456' },
-].map(author => {
+];
+
+export default () => authors.map(author => {
```

Comment:
do we really need this separation? it seems like `const authors` are not using anywhere